api/backup.go

```diff
@@ -122,8 +122,15 @@ func (s *Server) BackupDelete(w http.ResponseWriter, req *http.Request) error {
 
 	volName := mux.Vars(req)["volName"]
 
-	if err := s.m.DeleteBackup(input.Name, volName); err != nil {
-		return errors.Wrapf(err, "error deleting backup %v of volume %v", input.Name, volName)
+	backup, err := s.m.GetBackup(input.Name, volName)
+	if err != nil {
+		logrus.Errorf("Error getting backup %v of volume %v: %v", input.Name, volName, err)
```

Comment:
Because you tolerate this error on purpose, please use `warnf` and `logrus.witherror`. 

Suggest changing the log to `failed to get backup %v of volume %v`.

Reply:
I have fixed it following the comments. Thank you.